cli/src/pcluster/config/cfn_param_types.py

```diff
@@ -31,6 +31,12 @@
     get_supported_architectures_for_instance_type,
 )
 
+EBS_VOLUME_TYPE_IOPS_DEFAULT = {
```

Comment:
nit: you could move this inside the VolumeIopsParam class, as a class constant

Reply:
Done